src/client/dfs/dfs.c

```diff
@@ -231,6 +231,38 @@ oid_cp(daos_obj_id_t *dst, daos_obj_id_t src)
 	dst->lo = src.lo;
 }
 
+static inline int
+check_tx(daos_handle_t th, int rc)
+{
+	/** if we are not using a DTX, no restart is possible */
+	if (daos_handle_is_valid(th)) {
+		int ret = rc;
```

Comment:
Is this assignment meaningful? ret is not read before line 255 assigns a new value to it.

Reply:
you are right.